pages/glossary/attribute-value.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Attribute value
+key: attribute-value
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+---
+
+The _attribute value_ of an attribute set on an HTML element is the value that the attribute gets after being parsed and computed according to specifications. It may differ from the value that is actually written in the HTML code due to trimming whitespace or non-digits characters, default values, or case-insensitivity.
+
+Some notable case of attribute value, among others:
+
+- For [enumerated attributes][], the _attribute value_ is either the state of the attribute, or the keyword that maps to it; even for the default states. Thus `<input type="image" />` has an attribute value of either `Image Button` (the state) or `image` (the keyword mapping to it), both formulation having the same meaning; similarly, "an input element with a `type` _attribute value_ of `Text`" can be either `<input type="text" />`, `<input />` (missing value default), or `<input type="invalid" />` (invalid value default).
+- For [boolean attributes][], the _attribute value_ is `true` when the attribute represents the true value and `false` otherwise. Thus `<button disabled>`, `<button disabled="disabled">` and `<button disabled="">` all have a `disabled` _attribute value_ of `true`.
```

Comment:
```suggestion
- For [boolean attributes][], the _attribute value_ is `true` when the presence of the attribute represents the true value and `false` otherwise. Thus `<button disabled>`, `<button disabled="disabled">` and `<button disabled="">` all have a `disabled` _attribute value_ of `true`.
```
I think this makes it easier to understand

Reply:
I don't think this would be correct.
The presence of the attribute **always** represent the true value (per definition), thus
> the attribute value is `true` when the presence of the attribute represents the truth value

is equivalent to
> the attribute value is always `true`

Reply:
Still, I have to say that I have troubles in understanding the sentence "the attribute value is `true` when the attribute represents the true value". What does it mean for an attribute to represent the true value? Does it have to be equal to true?

What about "the _attribute value_ is `true` when present and `false` otherwise"?

Reply:
Still, I have to say that I have trouble in understanding the sentence "the attribute value is `true` when the attribute represents the true value". What does it mean for an attribute to represent the true value? Does it have to be equal to true?

What about "the _attribute value_ is `true` when present and `false` otherwise"?

Reply:
Hmmm… 🤔 
Yeah, "represent" is weird but its the word used by the specs…
>  The presence of a boolean attribute on an element represents the true value, and the absence of the attribute represents the false value.

So, "the attribute represents the true value" is essentially equivalent to "the attribute is present".

I wanted to not copy the def from the spec but rather use the result… I guess maybe that wasn't such a great idea…